src/containers/ConfirmationValidatorApi/ConfirmationValidatorApiUpgradeRequest/index.tsx

```diff
@@ -47,12 +45,42 @@ const ConfirmationValidatorApiUpgradeRequest: FC = () => {
 }`}
         heading="Request"
       />
-      <RequestResponseSnippet code={`{}`} heading="Response" />
+      <RequestResponseSnippet
```

Comment:
If you are updating the response kindly update the request as well so that the common data like **node_identifier** is similar and doesn't create confusion.

Reply:
better yet if you can setup a node locally and use actual response instead of tests you can ask @VincentHch for help setting the env up or maybe he can test it out for you.

Reply:
I've updated the node_identifier to match the response - I will reach out to Vincent for help with setting up an environment for the future.